packages/extension-browser/scripts/submit-packages.js

```diff
@@ -0,0 +1,61 @@
+const pptr = require('puppeteer-core');
+const chromiumFinder = require('@hint/utils').chromiumFinder;
+
+const stores = [
+    'https://addons.mozilla.org/en-US/developers/addons',
+    'https://chrome.google.com/webstore/developer/dashboard'
+];
```

Comment:
This is cool, especially if we end up getting it to do more that launch the URL in the future.

That said, if we end up sticking with just opening the page we could probably use shell commands for the relevant OS to open the links in a user's default browser. 

E.g.
* macOS: `open "https://webhint.io"`
* Windows: `start https//webhint.io`

Reply:
We'll see how this goes and change later. `puppeteer` is already installed for this package and I'd like to avoid doing more changes to this now 😊